app/src/main/java/com/framgia/chat_03/screen/signin/SignInActivity.java

```diff
@@ -18,6 +22,7 @@
  * SignIn Screen.
  */
 public class SignInActivity extends BaseActivity implements SignInContract.View, View.OnClickListener {
+    public static final int REQUEST_CODE = 97;
```

Comment:
tên biến nên đặt rõ nghĩa hơn
```suggestion
    public static final int REQUEST_SIGN_UP = 97;
```